cli/sync.go

```diff
@@ -180,6 +188,10 @@ var SyncCheckBadCmd = &cli.Command{
 		defer closer()
 		ctx := ReqContext(cctx)
 
+		if cctx.NArg() != 1 {
```

Comment:
dupliates a block just below it, again you could use `IncorrectNumArgsWithHint` and bring it to the top if you like